examples/acoustic/operators.py

```diff
@@ -24,9 +24,9 @@ def ForwardOperator(model, source, receiver, time_order=2, space_order=4,
                  time_order=time_order, space_order=space_order, save=save,
                  dtype=model.dtype)
     src = PointSource(name='src', ntime=source.nt, ndim=source.ndim,
-                      npoint=source.npoint)
+                      npoint=source.npoint, coordinates=source.coordinates.data)
```

Comment:
@mlange05 what's the point of this two extra initialization, there is already a `source` and `receiver` in the input that can be directly used.